recipes-extended/packagegroups/packagegroup-rte.bb

```diff
@@ -32,6 +32,9 @@ RDEPENDS_packagegroup-rte-core = " \
     u-boot \
     openvpn \
     openvpn-rte \
+    wpa-supplicant \
```

Comment:
@Cezarus27 I would put `wpa-supplicant` into the utility packagegroup

Reply:
@TomaszAIR Done.